src/Bicep.Core/Syntax/ParameterDeclarationSyntax.cs

```diff
@@ -64,6 +65,8 @@ public TypeSymbol GetDeclaredType()
 
         public TypeSymbol GetAssignedType(ITypeManager typeManager, ArraySyntax? allowedSyntax)
         {
+            static bool IsSecure(SyntaxBase? value) => value is BooleanLiteralSyntax boolLiteral && boolLiteral.Value;
```

Comment:
What about `IsBooleanTrue` instead of `IsSecure`?

Reply:
modified.